playwright/.env.example

```diff
@@ -0,0 +1,17 @@
+# Browserstack
```

Comment:
Can Playwright not access the vars from the main env file?

Reply:
This is more for our reference when someone wants to run tests locally like on other repos.